packages/contracts-bedrock/src/universal/StandardBridge.sol

```diff
@@ -352,6 +426,126 @@ abstract contract StandardBridge is Initializable {
         // contracts may override this function in order to emit legacy events as well.
         _emitERC20BridgeInitiated(_localToken, _remoteToken, _from, _to, _amount, _extraData);
 
+        _sendFinalizeBridgeERC20Message(_localToken, _remoteToken, _from, _to, _amount, _extraData, _minGasLimit);
```

Comment:
I like how we share the code here, wondering if we could share even more of the code between this and the new path? seems like there's still a decent amount of overlap

Reply:
not sure if im interpreting this correctly but i also added a new `_increaseDeposits` and `_decreaseDeposits` to replace the `deposits[_localToken][_remoteToken] = deposits[_localToken][_remoteToken] + _amount;` that exists throughout

Reply:
lmk if you think we should do something similar for anything else